dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -264,8 +264,8 @@ def _AwaitFlow(self, client: Client, flow_id: str) -> None:
       try:
         status = client.Flow(flow_id).Get().data
       except grr_errors.UnknownError:
-        msg = 'Unable to stat flow {0:s} for host {1:s}'.format(
-            flow_id, client.data.os_info.fqdn.lower())
+        msg = (f'Unable to start flow {flow_id} for host '
```

Comment:
Drive-by comment - `stat` was not a typo here, we're trying to get info on the flow, not start it. "stat" comes from the error you get from the GRR API when the flow doesn't exist (similar to filesystem `stat`), IIRC.